tests/cmdline/params/types/test_computer.py

```diff
@@ -8,88 +8,110 @@
-            'transport_type': 'local',
-            'scheduler_type': 'direct',
-            'workdir': '/tmp/aiida'
-        }
-
-        cls.param = ComputerParamType()
-        cls.entity_01 = orm.Computer(label='computer_01', **kwargs).store()
-        cls.entity_02 = orm.Computer(label=str(cls.entity_01.pk), **kwargs).store()
-        cls.entity_03 = orm.Computer(label=str(cls.entity_01.uuid), **kwargs).store()
-
-    def test_get_by_id(self):
-        """
-        Verify that using the ID will retrieve the correct entity
-        """
-        identifier = f'{self.entity_01.pk}'
-        result = self.param.convert(identifier, None, None)
-        self.assertEqual(result.uuid, self.entity_01.uuid)
-
-    def test_get_by_uuid(self):
-        """
-        Verify that using the UUID will retrieve the correct entity
-        """
-        identifier = f'{self.entity_01.uuid}'
-        result = self.param.convert(identifier, None, None)
-        self.assertEqual(result.uuid, self.entity_01.uuid)
-
-    def test_get_by_label(self):
-        """
-        Verify that using the LABEL will retrieve the correct entity
-        """
-        identifier = f'{self.entity_01.label}'
-        result = self.param.convert(identifier, None, None)
-        self.assertEqual(result.uuid, self.entity_01.uuid)
-
-    def test_ambiguous_label_pk(self):
-        """
-        Situation: LABEL of entity_02 is exactly equal to ID of entity_01
-
-        Verify that using an ambiguous identifier gives precedence to the ID interpretation
-        Appending the special ambiguity breaker character will force the identifier to be treated as a LABEL
-        """
-        identifier = f'{self.entity_02.label}'
-        result = self.param.convert(identifier, None, None)
-        self.assertEqual(result.uuid, self.entity_01.uuid)
-
-        identifier = f'{self.entity_02.label}{OrmEntityLoader.label_ambiguity_breaker}'
-        result = self.param.convert(identifier, None, None)
-        self.assertEqual(result.uuid, self.entity_02.uuid)
-
-    def test_ambiguous_label_uuid(self):
-        """
-        Situation: LABEL of entity_03 is exactly equal to UUID of entity_01
-
-        Verify that using an ambiguous identifier gives precedence to the UUID interpretation
-        Appending the special ambiguity breaker character will force the identifier to be treated as a LABEL
-        """
-        identifier = f'{self.entity_03.label}'
-        result = self.param.convert(identifier, None, None)
-        self.assertEqual(result.uuid, self.entity_01.uuid)
-
-        identifier = f'{self.entity_03.label}{OrmEntityLoader.label_ambiguity_breaker}'
-        result = self.param.convert(identifier, None, None)
-        self.assertEqual(result.uuid, self.entity_03.uuid)
+@pytest.fixture
+def parameter_type():
+    """Return an instance of the `ComputerParamType`."""
+    return ComputerParamType()
+
+
+@pytest.fixture
+def setup_computers(clear_database_before_test):
```

Comment:
```suggestion
@pytest.mark.usefixtures('clear_database_before_test')
def setup_computers():
```

Reply:
Uhm, we do it in plenty of places with no problems. So I doubt that it is not possible. What makes you think that besides that PR you linked?